<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Mount</class>
 <widget class="QDialog" name="X2Mount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>766</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>766</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>766</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OnStep Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>4</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>9999</width>
       <height>9999</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="geometry">
       <rect>
        <x>281</x>
        <y>696</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>696</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>96</y>
        <width>430</width>
        <height>216</height>
       </rect>
      </property>
      <property name="title">
       <string>Time and Location</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>328</x>
         <y>64</y>
         <width>72</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Sync</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>64</y>
         <width>304</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Sync location,  timezone and time from TheSkyX :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>176</y>
         <width>136</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>date and time :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="time_date">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>176</y>
         <width>248</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>dd/mm/yy  -  hh:mm:ss</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>96</y>
         <width>128</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Longitude :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="longitude">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>96</y>
         <width>176</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>+XXX YY SS.s</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>120</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Latitude :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="latitude">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>120</y>
         <width>176</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>+XX YY SS.s</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>144</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Time Zone :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="timezone">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>144</y>
         <width>176</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>GMT +/-XX</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>32</y>
         <width>328</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Automatically sync time and location on connect</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>144</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Â© RTI-Zone.org, 2024</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_logo_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>116</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>OnStep.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="X2_PhotoFileName" stdset="0">
       <string>OnStep.png</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>430</width>
        <height>216</height>
       </rect>
      </property>
      <property name="title">
       <string>Mount settings</string>
      </property>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>168</x>
         <y>32</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Home mount</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>144</y>
         <width>200</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop tracking on disconnect</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>112</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Use TheSkyX sync to add allignement stars</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>96</x>
         <y>176</y>
         <width>184</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>0.25x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.5x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1x (Guide)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4x (Centering)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8x (Move)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24x (Slew)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>48x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Half-Max</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Max</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>176</y>
         <width>72</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Slew rate :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="homingProgress">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>56</y>
         <width>96</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Homing C</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>88</y>
         <width>112</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>230400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>460800</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>88</y>
         <width>128</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Connection Speed :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>88</y>
         <width>128</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>(as set in Config.h)</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>560</y>
        <width>430</width>
        <height>120</height>
       </rect>
      </property>
      <property name="title">
       <string>Parking</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>140</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Park position (Atl/Az):</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>240</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>+00Âº 00' 00.0&quot; / 270Âº 00' 00.0&quot;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>+00Âº 00' 00.0&quot; / 180Âº 00' 00.0&quot;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>+00Âº 00' 00.0&quot; / 090Âº 00' 00.0&quot;</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>64</y>
         <width>240</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Goto park position and set in mount</string>
       </property>
      </widget>
      <widget class="QLabel" name="parkingProgress">
       <property name="geometry">
        <rect>
         <x>48</x>
         <y>88</y>
         <width>192</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Slewing to new park position C</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>272</x>
         <y>64</y>
         <width>144</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Set to current position</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
